packages/services/src/models/notification-scanners/DisputeRuled.js

```diff
@@ -3,9 +3,9 @@ import Network from '@aragonone/court-backend-server/build/web3/Network'
 const OUTCOMES = {
   0: 'Missing',
   1: 'Leaked',
-  2: 'Refused',
-  3: 'Against',
-  4: 'InFavor',
+  2: 'Refuse to vote',
+  3: 'Block',
+  4: 'Allow',
```

Comment:
I wonder if these should be in past term, since they're for the ruling?

Reply:
I agree, changed!

Reply:
I disagree. The ruling is not "Allowed", it is "Allow" (and same for the other rulings). I see nothing wrong with a statement such as:

> The final ruling in Dispute #0 is: Allow

or if you are writing/speaking in the past tense:

> After two rounds of deliberation, in their final ruling jurors a final ruling to **Allow** the disputed action.

Reply:
Hmm but it's also past in the dashboard itself so if we change we should do it everywhere at the same time.
![image](https://user-images.githubusercontent.com/26359933/83902512-c9a3b380-a75c-11ea-8598-dde62d4d88b0.png)

Reply:
![image](https://user-images.githubusercontent.com/26359933/83902593-f5269e00-a75c-11ea-89be-02bcf81c633c.png)

Reply:
I did not notice that before but if that's the case then yes I think we should be consistent about what the ruling outcomes are and keep the rulings in the present tense ("Allow", "Block", "Refuse to vote")